### ISSUE TITLE
Consider alternate docker images for development on apple chips

### ISSUE THREAD

**Issue by sgfost** [virtualcommons/port-of-mars#885]
build/response times are incredibly slow in environments running on arm64

https://hub.docker.com/r/arm64v8/node/

**Comment by alee** [comment]
was this a problem last year as well? @sabrinanel3 also has a mac but I'm not sure if she's also running into similar issues...

I assume this is running the latest versions of Docker Engine for Mac?

**Comment by sgfost** [comment]
Yeah both Sabrina and @saachibm are/have been dealing with it. Upgrading may be something to try first since one would imagine this would be a common issue

**Comment by alee** [comment]
another option could be to set up Linux on the Mac via something like https://mac.getutm.app/

**Comment by sgfost** [comment]
changing the image to `arm64v8/node:16-bullseye-slim` in the server dockerfile works and speeds things up significantly, doing the same for the client fails for some reason, need to investigate more
